awsdriver/service/vpc_cf.py

```diff
@@ -70,12 +73,14 @@ def create(self, resource_id, lifecycle_name, driver_files, system_properties, r
             try:
                 stack_id = cloudformation_driver.create_stack(stack_name, cf_template, cf_parameters)
                 logger.debug(f'Created Stack Id: {stack_id}')
+                self.__wait_for_transitgateway_availability(aws_location)
```

Comment:
This should be before stack creation as throwing after stack creation will leave the stack in cloud without automation handler and unable to clean up after that